tests/ignite/contrib/handlers/test_time_profilers.py

```diff
@@ -289,8 +289,10 @@ def delay_start(engine):
 
     assert "delay_start" in event_results[0]
     assert event_results[1] == "STARTED"
-
-    assert event_results[2] == approx(true_event_handler_time, abs=1e-1)  # total
+    if isinstance(event_results[2], str):
```

Comment:
I would say this is strange as assertion as `STARTED` event should be always triggered if we run `dummy_trainer`.

Reply:
Okay sure, Let me take a deeper look to see if I find anything insightful.